cypress/src/tests/e2eTests/verifyGuestUserCheckout.spec.js

```diff
@@ -25,10 +25,15 @@ import {
 
 describe('Verify guest user can place order', () => {
     it('Verify guest user can place order', () => {
+        Cypress.on('uncaught:exception', (err, runnable) => {
+            return false;
+        });
         cy.visit('');
         cy.get('.nav-drop')
             .contains('Catalog')
             .click();
+        // Randomly empty click is triggered 
```

Comment:
wdym randomly?

Reply:
https://github.com/hlxsites/aem-boilerplate-commerce/actions/runs/11219937102/job/31187005036 guest checkout failed twice. I could see this on my local where cypress was clicking on product link too fast that empty click happened so, cypress was expecting PDP page but app is still on home page. After adding this wait, i have not reproduced it.

Reply:
I like the format for comments like `// X because Y

`// adding a wait because product link is clicked too quickly`

But more importantly, you could probably avoid this by waiting for some specific element that exists on pdp to appear, rather than an arbitrary wait.